tests/tabs.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { test, expect } from '@playwright/test';
+import { waitForCustomEvent } from './lib/helper.ts'
+
+test('Tabs E2E Test', async ({ page }) => {
+  await page.goto('/test.html');
+
+  // Confirm All Tabs Render Correctly
+  const selectedTestTabs = page.getByTestId("Selected Test")
+
+  const tab1 = page.getByRole('button', { name: 'Tab 1' })
+  const tab2 = page.getByRole('button', { name: 'Tab 2' })
+  const tab3 = page.getByRole('button', { name: 'Tab 3' })
+
+  // Check Tabs Buttons Are Visible
+  await expect(selectedTestTabs).toBeVisible()
+
+  await expect(tab1).toBeVisible()
+  await expect(tab2).toBeVisible()
+  await expect(tab3).toBeVisible()
+
+  // Check Tab Buttons Have Correct Text
+  await expect(tab1).toHaveText('Tab 1')
+  await expect(tab2).toHaveText('Tab 2')
+  await expect(tab3).toHaveText('Tab 3')
+
+  // Check Selected Attribute is Correct (Tab 2)
+  await expect(selectedTestTabs).toHaveAttribute("selected", "Tab 2")
+
+  // Check Selected Tab Has Correct Background Color of White (Tab 2)
+  await expect(tab1).not.toHaveCSS("background-color", "rgb(255, 255, 255)")
+  await expect(tab2).toHaveCSS("background-color", "rgb(255, 255, 255)")
+  await expect(tab3).not.toHaveCSS("background-color", "rgb(255, 255, 255)")
+
+  // Click on Tab 3 
+  // Check That New Selected Tab is Correct (Tab 3)
+  const tab3Selected = waitForCustomEvent(page,'input')
+  await tab3.click()
+  await expect(tab3Selected).toBeTruthy()
+
+  // Check That New Selected Tab Has Correct Background Color of White (Tab 3)
+  await expect(tab1).not.toHaveCSS("background-color", "rgb(255, 255, 255)")
+  await expect(tab2).not.toHaveCSS("background-color", "rgb(255, 255, 255)")
+  await expect(tab3).toHaveCSS("background-color", "rgb(255, 255, 255)")
+
+  // Check That If No Selected Value, Default Selected Is First Value (Tab A)
```

Comment:
In regard to the default test, currently v-tabs has no default behavior. 
 When no selected attribute has been defined, no tab is rendered as selected.  The test asks that the first tab be selected.  Should I add this behavior into tabs.svelte?

Reply:
I think I disagree with that behavior. We can discuss with devin when he gets back but I think that nothing being selected is both useful and less surprising / trying to be less magical when there's no selected attribute. I'd instead test the current behavior for now.

Reply:
@DTCurrie - just wanted to check if you had an opinion on v-tabs not having default behavior.  The ticket asks for a default test which shows the first tab as selected when no selected tab is specified.  Currently, that behavior doesn't exist in v-tabs.

Reply:
Per an offline discussion with DC, he confirmed it's fine to remove the default behavior test from the ticket.